### PR TITLE
#4526 [Manager] Refactor the Files page to use the Material-Table

### PR DESCRIPTION
**Pull request by eechih**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4526 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Workspace/Edit/File/FileTableStyles.js [hunk 1]
```diff
@@ -16,13 +16,4 @@
...
-  ({ theme }) => css`
-    .actions {
-      margin-bottom: ${theme.spacing(3)}px;
-    }
-    .checkIcon {
-      color: ${theme.palette.success.main};
-    }
-  `,
-);
+export const Wrapper = styled.div(({ theme }) => css``);
```
Comment (konekoya) [inline ohara-manager/client/src/components/Workspace/Edit/File/FileTableStyles.js:19]: I think this file could be remove as we don't have any styles for this :)

File: ohara-manager/client/src/components/Workspace/Edit/File/FileTable.js [hunk 2]
```diff
@@ -14,64 +14,76 @@
...
-            </TableRow>
-          );
-        })}
-      </Table>
-    </Wrapper>
+    <>
+      <MuiTable
+        actions={[
+          {
+            icon: () => <AddBox />,
```
Comment (konekoya) [inline ohara-manager/client/src/components/Workspace/Edit/File/FileTable.js:37]: Just curious if `icon: <Addbox />` works in this case?

File: ohara-manager/client/src/components/Workspace/Edit/File/FileTable.js [hunk 3]
```diff
@@ -14,64 +14,76 @@
...
+            tooltip: 'Upload File',
+            isFreeAction: true,
```
Comment (konekoya) [inline ohara-manager/client/src/components/Workspace/Edit/File/FileTable.js:39]: What does this mean 🤔

Comment (eechih) [reply ohara-manager/client/src/components/Workspace/Edit/File/FileTable.js:39]: isFreeAction => Independent actions that will not on row' actions section

https://material-table.com/#/docs/all-props

File: ohara-manager/client/src/components/Workspace/Edit/File/FileTable.js (hunk 2 above)
Comment (eechih) [reply ohara-manager/client/src/components/Workspace/Edit/File/FileTable.js:37]: icon 只接受 String 及 Function

File: ohara-manager/client/src/components/Workspace/Edit/File/FileTableStyles.js (hunk 1 above)
Comment (eechih) [reply ohara-manager/client/src/components/Workspace/Edit/File/FileTableStyles.js:19]: I will remove it, thanks :)

File: ohara-manager/client/src/components/Workspace/Edit/File/FileTable.js (hunk 3 above)
Comment (konekoya) [reply ohara-manager/client/src/components/Workspace/Edit/File/FileTable.js:39]: Sooorry, thought it was our API XD

**Pull request merged**